test/built-ins/GeneratorPrototype/return/try-finally-set-property-within-try.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2022 Bo Pang. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-generator.prototype.return
+description: >
+    When a generator is paused within a `try` block of a `try..finally`
+    statement, `return` should interrupt control flow as if a `return`
+    statement had appeared at that location in the function body.
+    The `finally` block is still evaluated, and may override the return value.
+features: [generators]
+---*/
+
+
+var obj={foo:undefined};
```

Comment:
As a suggestion to avoid false positives, replace the undefined value with something more specific like:

```suggestion
var obj={foo:'not modified'};
```